phpunit.xml.dist

```diff
@@ -13,6 +13,7 @@
   </coverage>
   <php>
     <ini name="error_reporting" value="-1"/>
+    <env name="APP_ENV" value="test" force="true" />
```

Comment:
Normlane `<env` v configu PHPunitu nemame, ale tady se mi nejak nedarilo ho presvedcit. Mozna to souvidi s tim, ze tu `APP_ENV` mame nastaveny uz `Dockerfile` :shrug: